### DISCUSSION TITLE
Customise the window title prefix/postfix

### DISCUSSION THREAD

**Discussion by frasty** [l10nelw/winger#7] [Ideas]
Hi,
To differentiate among multiple Firefox profiles, window titler extension allows to add a different window prefix that belongs to each Firefox profile. That way I always know in which profile I'm currently in. Would be nice if winger could inherit that window prefix or, broadly speaking, inherit part of the window title before the first " - ".
Thanks for the efforts you put on this addon

**Comment by Kookiness** [comment]
Similar to this idea, I was just wondering if it might be possible to have the option to add a custom prefix to all of our Winger-named windows.

It could help the named windows stand out while looking through a list of multiple Firefox windows when mousing over the Firefox icon in the taskbar.

I'm commenting here rather than starting a new thread, because the ideas seemed similar enough that they might be able to be added as one feature!

I could add a custom prefix (e.g. "> " is what I like to use) to the window names individually myself, but unfortunately,
- This would be less convenient as it needs to be done for every window, and even less convenient if we want to change our mind about what our custom prefix should be
- This would remove the ability in context menus to choose "Send to Window" and then use the keyboard to press the first letter of a named window to quickly jump to in the list

I love all the work you've done on this extension! It's my favourite extension for moving tabs around. :D Thank you!

**Comment by l10nelw** [comment]
I'll implement this by allowing custom name prefix and postfix (per profile), meaning instead of the default `name -` you could have `[name]` or `>name ~` for instance.

I'm releasing a new version soon, so planning to meet this request in the next one after.

**Reply by Kookiness** [reply]
Fantastic, thank you so much!

**Comment by l10nelw** [comment]
Hi @Kookiness, this is now possible with the latest version 2.4.0. Sorry for the wait!

**Reply by Kookiness** [reply]
Awesome, thank you so much, I'll give it a try!
